packages/cli/create-strapi-app/utils/sanitize-input.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const quote = require('shell-quote').quote;
+
+module.exports = input => {
```

Comment:
I think it would be better to export a named function, it helps IDEs auto-import the functions

Reply:
ok sounds good I'll update